### PR TITLE
Combine BraintreeUIKit & BraintreeDropIn frameworks

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

 - Move all files from BraintreeUIKit into BraintreeDropIn
 - Update header imports accordingly
 - Other updates:
    - Rakefile
    - GH Actions release.yml
    - SPMTest app
    - Demo app
    - Umbrella header used for docs generation
    - Podspec & Package.swift

 ### Checklist

 - [X] Added a changelog entry

### Authors
@scannillo @sestevens

### CODE REVIEW

File: Demo/Demo Base/DemoDropInViewController.swift [hunk 1]
```diff
@@ -144,13 +141,9 @@ class DemoDropInViewController: DemoBaseViewController {
 
     func updatePaymentMethodNonce(_ result: BTDropInResult?) {
         demoView.paymentMethodTypeLabel.isHidden = (result?.paymentMethod == nil)
-        demoView.paymentMethodTypeIcon.isHidden = (result?.paymentMethod == nil)
-        if let nonce = result?.paymentMethod {
-            let paymentMethodType = BTUIKViewUtil.paymentOptionType(forPaymentInfoType: nonce.type)
```
Comment (scannillo) [inline Demo/Demo Base/DemoDropInViewController.swift:149]: We updated the Demo app to no longer use the `BTUIKViewUtil` to fetch the payment method icon view.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Welcome to Braintree's Drop-In SDK for iOS!
 
 **The Braintree iOS Drop-In SDK requires Xcode 12+**. It permits a deployment target of iOS 12.0 or higher.
 
+<!--TODO: Update README for v9 major version changes -->
```
Comment (sestevens) [inline README.md:15]: We'll address this in a separate PR

File: Sources/BraintreeDropIn/BTPaymentMethodNonce+DropIn.h [hunk 3]
```diff
@@ -8,6 +8,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface BTPaymentMethodNonce (DropIn)
 
+// TODO - Move this to BTDropInResult; when Apple Pay is selected on Drop-In, nonce is null, so there is never a description.
```
Comment (sestevens) [inline Sources/BraintreeDropIn/BTPaymentMethodNonce+DropIn.h:11]: This will also be a separate PR

**Review by sshropshire — APPROVED** [review]
🌊 &nbsp;This is awesome!

**Review by sarahkoop — APPROVED** [review]

**Review by demerino — APPROVED** [review]
